OWML.Launcher/App.cs

```diff
@@ -138,7 +138,7 @@ private void OnOutput(string s)
 
         private bool HasVrMod(IList<IModData> mods)
         {
-            var vrMod = mods.FirstOrDefault(x => x.Config.RequireVR && x.Config.Enabled);
+            var vrMod = mods.FirstOrDefault(x => x.Manifest.RequireVR && x.Config.Enabled);
```

Comment:
...how about also checking config here? (see other comment)

Reply:
Maybe it's better to add RequireVR to ModData then?